presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -151,3 +151,8 @@ Hive, this table can be described in Presto::
 This table can then be queried in Presto::
 
     SELECT * FROM hive.web.page_view;
+
+Hive Connector Limitations
+---------------------------
+
+* :doc:`/sql/delete` : Only supports ``DELETE`` where one or more partitions are deleted entirely using condition on partitioning columns.
```

Comment:
The reason I had to phrase it like this is due to the limitations that if the where clause is based on a function of partitioning key then it does not delete even if it matches the entire partitions. Here is an example,
``
presto:default> show partitions from t1_p;
     c2     
------------
 1992-01-01 
 2013-02-01 
 2020-02-03 
(3 rows)

presto:default> select month(cast(c2 as date)) from t1_p;
 _col0 
-------
     2 
     1 
     2 
presto:default> delete from t1_p where month(cast(c2 as date))= 1;
Query 20160403_224232_00011_ppfuv failed: This connector only supports delete where one or more partitions are deleted entirely
``